torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -275,6 +275,12 @@ class GpuLower::KernelIrMapper : private OptInConstDispatch {
     TORCH_CHECK(gpu_lower_->kir_expr_map_.insert({node, lowered_node}).second);
   }
 
+  void handle(const TransposeOp* node) final {
+    const auto lowered_node = ir_builder_.create<kir::TransposeOp>(
+        lowerValue(node->out()), lowerValue(node->in()));
```

Comment:
is it ok to discard the old2map here?

Reply:
At this point, it isn't used, so should be ok.